Documentation/Blazorise.Docs/Components/Footer.razor

```diff
@@ -103,4 +104,26 @@
             </UnorderedList>
         </Column>
     </Row>
-</Container>
\ No newline at end of file
+</Container>
+@code{
+    [Inject] private ThemeService ThemeService { get; set; }
+
+    private TextColor IconTextColor => ThemeService.CurrentTheme == "Dark" ? TextColor.Light : TextColor.Dark;
+    
+    protected override void OnInitialized()
+    {
+        ThemeService.ThemeChanged += OnThemeChanged;
+
+        base.OnInitialized();
+    }
+
+    public void Dispose()
+    {
+        ThemeService.ThemeChanged -= OnThemeChanged;
+    }
+
+    async void OnThemeChanged( object sender, string theme )
```

Comment:
If you do StateHasChanged in app.razor, is that not enough to update all the underlying components?

Reply:
I have removed code from App.razor. It didn't work